compiler/optimizer/OMROptimizationManager.cpp

```diff
@@ -49,7 +49,7 @@ OMR::OptimizationManager::OptimizationManager(TR::Optimizer *o, OptimizationFact
         _numPassesCompleted(0),
         _optData(NULL),
         _optPolicy(NULL),
-        _enabled(false),
+        _enabled(!o->comp()->isDisabled(optNum)),
```

Comment:
Wow, how awkward. In light of the above, I agree with the approach of just trying to make it mirror the current enable/disable state in light of the above discussion (the ramifications of the regex did not dawn on me until the above discussion).